test/auto/parser_spec.lua

```diff
@@ -257,6 +258,20 @@ describe("parse", function()
     end)
   end)
 
+  describe("[select]", function()
+    it('works with [select] = "a single string"', function()
+      local defs = {
+        select = {
+          "id",
+          z = fn "date('now')",
```

Comment:
So this is now the only way to execute something as sql?. 🤔 I don't mind it

Reply:
If you use bindings - yes. But you can always use raw: `insert date('now') into ...`.